packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -117,7 +118,14 @@ const Redirect = ({
             break;
         }
         case 'payment_withdraw': {
-            history.push(routes.cashier_withdrawal);
+            if (loginid.toLowerCase().includes('w')) {
```

Comment:
Treading on thin ice 😅

Reply:
i had discussion with @farzin-deriv, and seems like it's the only way we can distinguish CR and Wallet accounts

Reply:
@heorhi-deriv Maybe putting it in a named variable so at least later we know what does it mean 🤔 

```ts
const is_wallet_account = loginid.toLowerCase().includes('w');

if (is_wallet_account) {
```